### DISCUSSION TITLE
Does Floating UI works with NextJS?

### DISCUSSION THREAD

**Discussion by Harukisatoh** [floating-ui/floating-ui#2803] [Q&A]
I followed the [positioning tutorial](https://floating-ui.com/docs/react#positioning) on the docs, but I got this error:

<img width="1430" alt="image" src="https://github.com/floating-ui/floating-ui/assets/37125288/8e469081-53f8-4fc7-8dbd-2a21195ab0f0">

I'm currently using NextJS 14, and I'm trying to use the lib on a client component.

**Comment by atomiks** [comment]
Yes it works in client components fine. I'm not sure how that error would be appearing.

**Comment by Harukisatoh** [comment]
Nevermind, I'm stupid. The problem was on another component
